models/cards.js

```diff
@@ -1695,6 +1695,23 @@ if (Meteor.isServer) {
       const oldvalue = doc[action] || '';
       const activityType = `a-${action}`;
       const card = Cards.findOne(doc._id);
+      const list = card.list();
+      if (list) {
+        // change list modifiedAt
+        const modifiedAt = new Date();
+        const boardId = list.boardId;
+        Lists.direct.update(
+          {
+            _id: list._id,
+          },
+          {
+            $set: {
+              modifiedAt,
```

Comment:
I'm not sure what this does, sorting should not make changes to modified dates

Reply:
this allows when user has chosen an EndAt date of a list, but didn't want to archive the list, we could change the list modifiedAt date to affect the sort order instead of pulling cards in each list to do a hug comparision, which is too costy.